src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -157,10 +159,17 @@ namespace webnn_native { namespace ie {
             return activations;
         }
 
-        // Transpose NHWC <=> NCHW.
-        ngraph_node_t* TransposeInputLayout(const ngraph_node_t* input, bool nhwc_to_nchw) {
-            std::vector<int64_t> order =
-                nhwc_to_nchw ? std::vector<int64_t>{0, 3, 1, 2} : std::vector<int64_t>{0, 2, 3, 1};
+        // Transpose InputLayout(NHWC/HWNC) <=> NCHW.
+        ngraph_node_t* TransposeInputLayout(const ngraph_node_t* input,
```

Comment:
Can we remove the `bool direction` argument?

// inputLayout is destination layout.
```
ngraph_node_t* TransposeInputLayout(const ngraph_node_t* input, InputLayout inputLayout) {
   switch(inputLayout) {
       case: InputLayout::Nchw:
           order = {0, 3, 1, 2};
           break;
      case InputLayout::Nhwc:
           order = {0, 2, 3, 1};
           break;
      case InputLayout::Hwnc:
          order = {2, 3, 0, 1};
           break;
      default:
           DAWN_ASSERT(0);

   }
}
```

Reply:
NchwToNhwc:
    break;
NhwcToNchw:
    break;
HwncToNchw:
NchwToHwnc:
    order = {2, 3, 0, 1};
     break;

Reply:
Optimized the code. @fujunwei PTAL, thanks!